Vagrantfile

```diff
@@ -118,6 +118,10 @@ playground = {
     :mem => base_box_builder_mem,
     :net_auto_config => true,
     :show_gui => false,
+    :host_vars => {
```

Comment:
I removed this in #45, but I don't explicitly remember why. It was giving issues during provisioning. Can you please check if the base box provisioning works with this?

Reply:
yes it work properly! i am aware that the only `assigned_hostname` in the host_vars for the base-box will give issues, but i fixed the issue as well inside the condition.